metaci/build/models.py

```diff
@@ -534,26 +536,16 @@ def run_flow(self, project_config, org_config):
             raise FlowNotFoundError('Flow not found: {}'.format(self.flow))
         flow_config = FlowConfig(flow)
 
-        if settings.METACI_FLOW_SUBCLASS_ENABLED:
-            class_path = 'metaci.build.flows.MetaCIFlow'
-        else:
-            # Get the class to look up options
-            class_path = flow_config.config.get('class_path',
-                                                'cumulusci.core.flows.BaseFlow')
```

Comment:
Is it okay to drop this?

Reply:
yep, we don't support flow_config.class_path anymore.